lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -52,6 +52,9 @@
       description: controls verification and validation of SSL certificates, mostly you only want to turn off with self signed ones.
       type: boolean
       default: True
+    namespace:
+      description: namespace where secrets reside. requires HVAC 0.7.0+ and Vault 0.11+
+      default: None
```

Comment:
missing version_added

Reply:
what needs it? the documentation portion?

Reply:
https://github.com/ansible/ansible/blob/8b871bc070b4d761702182d60b541e864297d4c4/lib/ansible/plugins/lookup/hashi_vault.py#L11

Reply:
just looking at how this [dict is rendered on the screen](https://docs.ansible.com/ansible/latest/plugins/lookup/hashi_vault.html), it doesnt seem like it would know what to do with the version_added param under an option.